src/index.html

```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+
+  <body class="calendar">
+    <div class="calendar__field calendar__field--start-day--tue calendar__field--month-length--29">
```

Comment:
```suggestion
  <body>
    <div class="calendar calendar__field--start-day--tue calendar__field--month-length--29">
```
I guess this one is calendar by itself